tests/test_oc4ids_tranforms.py

```diff
@@ -0,0 +1,1332 @@
+                {"id": "GB-GOV-23", "name": "Department for Transport", "details": "This is also a test."},
+            ],
+            "planning": {
+                "budget": {
+                    "id": "1",
+                    "description": "Multi-source budget, see budget breakdown for details.",
+                    "amount": {"amount": 300000, "currency": "GBP"},
+                    "budgetBreakdown": [
+                        {
+                            "sourceParty": {
+                                "id": "GB-LAC-E09000003-557",
+                                "name": "London Borough of Barnet - Transport Services",
+                            },
+                            "period": {"startDate": "2016-01-01T00:00:00Z", "endDate": "2016-12-31T23:59:59Z"},
+                            "id": "001",
+                            "description": "Budget contribution from the local government",
+                            "amount": {"amount": 150000, "currency": "GBP"},
+                        },
+                        {
+                            "sourceParty": {"id": "GB-GOV-23", "name": "Department for Transport"},
+                            "period": {"startDate": "2016-01-01T00:00:00Z", "endDate": "2016-12-31T23:59:59Z"},
+                            "id": "002",
+                            "description": "Budget contribution from the national government",
+                            "amount": {"amount": 150000, "currency": "GBP"},
+                        },
+                    ],
+                }
+            },
+        }
+    ]
+
+    output = transforms._run_transforms(copy.deepcopy(releases), "1", transforms=[transforms.funding_sources],)
+
+    assert output["parties"][0]["id"] == "GB-LAC-E09000003-557"
+    assert output["parties"][0]["details"] == "This is just a test."
+    assert "funder" in output["parties"][0]["roles"]
+
+
+def test_funders():
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "parties": [
+                {
+                    "id": "GB-LAC-E09000003-557",
+                    "name": "London Borough of Barnet - Transport Services",
+                    "details": "This is just a test.",
+                    "roles": ["funder"],
+                },
+                {
+                    "id": "GB-GOV-23",
+                    "name": "Department for Transport",
+                    "details": "This is also a test.",
+                    "roles": ["funder"],
+                },
+            ],
+        }
+    ]
+
+    output = transforms._run_transforms(copy.deepcopy(releases), "1", transforms=[transforms.funding_sources],)
+
+    assert output["parties"][0]["id"] == "GB-LAC-E09000003-557"
+    assert output["parties"][0]["details"] == "This is just a test."
+    assert "funder" in output["parties"][0]["roles"]
+
+
+def test_cost_estimate():
```

Comment:
Should this transform work on `date` rather than the ordering of the input releases?

Also, should the test cover an input which has a subsequent 'tender' release with `tender.status` = 'active' and an updated `tender.value` to make sure this is not copied across?